Packs/EclecticIQ/Integrations/EclecticIQv2/EclecticIQv2.py

```diff
@@ -0,0 +1,640 @@
+            final_data = entity_data
+    human_readable_title = 'EclecticIQ observable reputation - {}'.format(
+        value_eiq)
+    human_readable = tableToMarkdown(human_readable_title, final_data)
+    context['Entity'] = createContext(
+        data=final_data, removeNull=True)
+    context[outputPaths['ip']] = standard_observable_outputs
+    return CommandResults(
+        readable_output=human_readable,
+        outputs_prefix='EclecticIQ',
+        outputs_key_field='value',
+        outputs=context
+    )
+
+
+def EclecticIQ_create_sighting(client: Client, args: Any) -> CommandResults:
+    """create_sighting command: Returns the sighting data
+    :type client: ``Client``
+    :param Client: EclecticIQ client to use
+    :type args: ``Any``
+    :param args: args {value, description, title, tags, type, confidence_level}
+    :return: sighting data
+    :rtype: ``CommandResults``
+    """
+    value = args.get("value")
+    description = args.get("description")
+    title = args.get("title")
+    tags = args.get("tags")
+    type_eiq = args.get("type")
+    confidence_level = args.get("confidence_level")
+    if not validate_type(type_eiq, value):
+        raise ValueError("Type does not match specified value")
+    response = client.sighting(
+        value, description, title, tags, type_eiq, confidence_level)
+    context = {}
+    output = {'value': value,
+              'description': description,
+              'title': title,
+              'tags': tags,
+              'Type': type_eiq,
+              'confidence_level': confidence_level}
+    human_readable_title = '!sighting created for- {}'.format(
+        args.get("value"))
+    human_readable = tableToMarkdown(human_readable_title, t=output)
+    context['Sighting.Data'] = createContext(
+        data=response, removeNull=True)
+    return CommandResults(
+        readable_output=human_readable,
+        outputs_prefix='Sighting.Data',
+        outputs_key_field='value',
+        outputs=output
+    )
+
+
+def EclecticIQ_create_observable(client: Client, args: Any) -> CommandResults:
+    """create_observable command: Returns the observable data
+    :type client: ``Client``
+    :param Client: EclecticIQ client to use
+    :type args: ``Any``
+    :param args: args {type, value}
+    :return: observable data
+    :rtype: ``CommandResults``
+    """
+    type_eiq = args.get("type")
+    value = args.get("value")
+    maliciousness = args.get("maliciousness")
+    if not validate_type(type_eiq, value):
+        raise ValueError("Type does not match specified value")
+    response = client.observable(type_eiq, value, maliciousness)
+    context = {}
+    output = {'type': type_eiq,
+              'value': value,
+              'maliciousness': maliciousness
+              }
+    human_readable_title = "Observables created successfully..!!"
+    human_readable = tableToMarkdown(human_readable_title, t=output)
+    context['Observable.Data'] = createContext(
```

Comment:
you are not using context, so Im not sure what you tried to do here.

Reply:
Updated